youtube_dl/extractor/bandcamp.py

```diff
@@ -47,6 +47,15 @@ def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
         title = mobj.group('title')
         webpage = self._download_webpage(url, title)
+        album = self._search_regex(
+            r'(?ms).*?title\s*?:\s*?"(?P<album>.*?)",',
```

Comment:
This extracts first title found not necessarily an album title.
All captured groups must be at least one character length.